lib/agenda/stop.js

```diff
@@ -18,8 +18,21 @@ module.exports = function(cb) {
   const _unlockJobs = function(done) {
     debug('Agenda._unlockJobs()');
     const jobIds = self._lockedJobs.map(job => job.attrs._id);
-    debug('about to unlock jobs with ids: %O', jobIds);
-    self._collection.updateMany({_id: {$in: jobIds}}, {$set: {lockedAt: null}}, done);
+
+    if (jobIds.length >= 1) {
```

Comment:
Length should always be compared. If length is missing it'll return undefined. I'll remove the unneeded return though, added this after being up 48+ hours. Oops.

```console
> !undefined
true
```